Frontend/ui-customer/src/Screens/RentalBookingFlow/RentalScreen/View.purs

```diff
@@ -0,0 +1,260 @@
+rentalVariantSelectionView push state =
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , background Color.white900
+    ]
+    [ linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , padding $ Padding 16 16 16 16
+        ]
+        [ textView $
+          [ text $ getString RENTAL_OPTIONS
+          ] <> FontStyle.h1 TypoGraphy
+        , separatorView push state
+        ]
+    , scrollView
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        ]
+        [ linearLayout
+          [ height WRAP_CONTENT
+          , width MATCH_PARENT
+          , margin $ Margin 16 16 16 32
+          , orientation VERTICAL
+          ]
+          ( map
+            ( \item ->
+                ChooseVehicle.view (push <<< ChooseVehicleAC) item
+            )
+            state.data.rentalBookingData.quoteList
+          )
+        ]
+    , linearLayout
+      [ height MATCH_PARENT
+      , width MATCH_PARENT
+      , weight 0.0
+      , gravity BOTTOM
+      , alignParentBottom "true, -1"
+      ]
+      [ linearLayout
+          [ height WRAP_CONTENT
+          , stroke $ "1," <> Color.grey900
+          , padding $ Padding 16 16 16 16
+          ] [PrimaryButton.view (push <<< PrimaryButtonActionController) (primaryButtonConfig state)]
+      ]
+    ]
+
+fareBreakupView :: forall w. (Action -> Effect Unit) -> RentalScreenState -> PrestoDOM (Effect Unit) w
+fareBreakupView push state =
+  linearLayout
+    [ height MATCH_PARENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    , background Color.white900
+    ]
+      [ GenericHeader.view (push <<< GenericHeaderAC) (genericHeaderConfig state)
+      , separatorView push state
+      , linearLayout
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , margin $ Margin 16 16 16 32
+        ]
+        ( map
+            ( \item ->
+                if (item.index == item.activeIndex) then ChooseVehicle.view (\action -> pure unit) (item) else emptyTextView
+            )
+            state.data.rentalBookingData.quoteList
+        )
+      , linearLayout 
+        [ height WRAP_CONTENT
+        , width MATCH_PARENT
+        , orientation VERTICAL
+        , margin $ MarginHorizontal 16 16
+        ]
+        (map (\item -> 
+          descriptionView push state (item)
+        ) [BookingTime, BookingDistance, BaseFare, TollFee])
+      , noteAndPrimaryButtonView push state
+      ]
+
+descriptionView :: forall w. (Action -> Effect Unit) -> RentalScreenState -> FareBreakupRowType -> PrestoDOM (Effect Unit) w
+descriptionView push state description =
+  linearLayout
+    [ height WRAP_CONTENT
+    , width MATCH_PARENT
+    , orientation VERTICAL
+    ]
+    [ textView $
+        [ width MATCH_PARENT
+        , height WRAP_CONTENT
+        , text case description of
```

Comment:
Can we move this case match to where clause and the get the String for both the text views